### ISSUE TITLE
Offline Mode: Service Page Not Working

### ISSUE THREAD

**Issue by phorne-uncharted** [microsoft/community-organization-operations-suite#615]
When in offline mode, the service page is not working properly. The user currently needs to navigate to the service page before going offline. As soon as the user navigates away from the service page when offline, they are unable to then go back to the service listing.

The application needs to be updated to properly store services when offline. The user should be able to go offline without navigating to the service page, and then be able to start a service while being offline. Furthermore, whenever the user navigates away from the service page, they should be able to go back and start a service all the while remaining offline.

**Steps to reproduce:**

1. Login
2. Go to Clients
3. Switch to offline mode
4. Go to Services

**Expected Result**
All services listed in the table

**Actual Result**
Table is empty, waiting on results to load

**Comment by jonmarcello** [comment]
Duplicate of #584
